src/components/DateRangePickerInput.jsx

```diff
@@ -123,6 +126,18 @@ export default class DateRangePickerInput extends React.Component {
     });
   }
 
+  renderInputIcon(inputIcon) {
+    return (<button
```

Comment:
the `(` should end the line, and the `)` should begin the line - iow:
```jsx
return (
  <button
    type="button"
    className="DateRangePickerInput__calendar-icon"
    disabled={this.props.disabled}
    aria-label={this.props.phrases.focusStartDate}
    onClick={this.props.onArrowDown}
  >
    {inputIcon}
  </button>
);
```

Also, please destructure all props at the top of the function.